src/index.html

```diff
@@ -1,15 +1,177 @@
+    <form
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+    >
+      <fieldset class="blocks">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            id="surname"
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            id="name"
+          >
+        </div>
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            value="12"
+            name="age"
+            min="1"
+            max="140"
+            id="age"
+          >
+        </div>
+        <div class="form-field">
+          <label for="date">Full date of birth:</label>
+          <input
+            type="date"
+            name="date"
+            id="date"
+          >
+        </div>
+        <div class="form-field">
+          <label for="terms">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="isTermAccepted"
+            id="terms"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="blocks">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            id="email"
+          >
+        </div>
+        <div class="form-field">
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="36"
+            required
+            id="pass"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="blocks">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            value="y"
+            id="catLove"
+          >
+          <label for="catLove">Yes</label>
+          <input
+            type="radio"
+            name="cats"
+            value="n"
+            id="catHate"
+          >
+          <label for="catHate">No</label>
+        </div>
+        <div class="form-field">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </div>
+        <div class="form-field">
+          <label for="bed">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="time"
```

Comment:
```suggestion
            name="bedtime"
```
be more specific with naming